client/components/domains/domain-upsell-callout/style.scss

```diff
@@ -0,0 +1,71 @@
+@import "@automattic/typography/styles/variables";
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+body.is-section-gutenberg-editor .domain-upsell-callout {
+	height: 60px;
+}
+
+.domain-upsell-callout {
+	font-size: $font-body-extra-small;
+	position: absolute;
+	z-index: 99998;
+	display: flex;
+	align-items: center;
+	left: 50%;
+	transform: translateX(-50%);
+
+	@media ( max-width: $break-xlarge ) {
+		display: none;
+	}
+
+	&__content {
+		width: 100%;
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		color: var(--studio-gray-60);
+	}
+
+	&__content-text {
+		display: flex;
+		align-items: center;
+		border-radius: 24px; /* stylelint-disable-line scales/radii */
+		padding: 6px 20px;
+		column-gap: 10px;
+		background: #f6f7f7;
+		@media ( max-width: $break-wide ) {
+			padding: 6px 10px;
+		}
+	}
+
+	&__button-text-desktop {
+		display: block;
+		@media ( max-width: $break-wide ) {
+			display: none;
+		}
+	}
+	&__button-text-mobile {
+		display: none;
+		@media ( max-width: $break-wide ) {
```

Comment:
Can we use this threshold to move it to the left as well? Especially in Portuguese the problem appears more as words get bigger. With `left: 42%;` instead `50%` in these sizes it seems to work better for me.

1450px works fine:

<img width="1587" alt="image" src="https://user-images.githubusercontent.com/1044309/217315014-f1929210-e135-4c0c-ab78-29e0a844b464.png">

1300px word cut:

<img width="1434" alt="image" src="https://user-images.githubusercontent.com/1044309/217315822-29d8e5b7-3f54-4792-bc66-f0698da9acec.png">

Even at the "mobile" breaker 1260, word cut:

<img width="1382" alt="image" src="https://user-images.githubusercontent.com/1044309/217316038-cbd05356-8b5d-4544-82e1-e9132d391978.png">

1080px:

<img width="1185" alt="image" src="https://user-images.githubusercontent.com/1044309/217316933-b9642396-2689-4082-bdce-441863370110.png">